src/CurrencyRateController.test.ts

```diff
@@ -63,4 +65,18 @@ describe('CurrencyRateController', () => {
 		expect(mock.called).toBe(true);
 		mock.restore();
 	});
+
+	it('should use default base asset', async () => {
+		const baseAsset = 'FOO';
+		const controller = new CurrencyRateController({ baseAsset });
+		const mock = stub(window, 'fetch');
+		(window.fetch as SinonStub).returns(
```

Comment:
Can we use `mock` here instead of casting `window.fetch`? `mock` should have `.returns` on it, I think.

Also we might be able to use `.resolves(x)` instead of `.returns(Promise.resolve(x))` here.